### DISCUSSION TITLE
Welcome to linux-voice-control Discussions!

### DISCUSSION THREAD

**Discussion by omegaui** [omegaui/linux-voice-control#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by MyLittleKony** [comment]
how can we install this on the steam deck?

here is the error i receive. would you consider a Flatpak?

cp: cannot create regular file '/usr/bin/linux-voice-control': Read-only file system
cp: cannot create regular file '/usr/bin/linux-voice-control-gui': Read-only file system
chmod: cannot access '/usr/bin/linux-voice-control': No such file or directory
chmod: cannot access '/usr/bin/linux-voice-control-gui': No such file or directory

**Reply by omegaui** [reply]
@MyLittleKony 
Yes, if your steam deck allows the `install.sh` script to write to the disk.
Which I can see clearly, its behaving as a read only fs.
